packages/lib/hooks/useTheme.tsx

```diff
@@ -6,12 +6,10 @@ import type { Maybe } from "@calcom/trpc/server";
 
 // makes sure the ui doesn't flash
 export default function useTheme(theme?: Maybe<string>) {
-  let currentTheme: Maybe<string> = theme || "system";
-
   const { resolvedTheme, setTheme, forcedTheme, theme: activeTheme } = useNextTheme();
   const embedTheme = useEmbedTheme();
   // Embed UI configuration takes more precedence over App Configuration
-  currentTheme = embedTheme || theme;
```

Comment:
It was overriding the previously set variable and thus default of "system" was never getting used.